content/compute/boot-from-volume.md

```diff
@@ -0,0 +1,199 @@
+/*
+Title: Booting an Instance from a Volume
+*/
```

Comment:
IMO, Booting should be at the bottom of the list -- you can use `Sort: X` in the header meta -- the higher the sort value, the heavier the menu item weight to sink.